helm/assets/configuration/spec.yaml

```
@@ -0,0 +1,12 @@
+name: helm
+files:
+- name: helm.yaml
+  options:
+  - template: init_config
+    options:
+    - template: init_config/default
+  - template: instances
+    options:
+    - template: instances/default
```

```suggestion
    - template: instances/tags
    - template: instances/service
    - template: instances/global
```
This should fix the CI config validation step.